indexing/src/test/java/com/google/enterprise/cloudsearch/sdk/indexing/template/ApiOperationsTest.java

```diff
@@ -128,10 +128,9 @@ public void pushItems_helpers() {
         .build();
     assertThat(pushItems.getPushItemResources().size(), equalTo(3));
     assertThat(pushItems.getPushItemResources().stream()
-        .map(item -> item.getId()).collect(ImmutableList.toImmutableList()),
-        equalTo(ImmutableList.of("item1", "item2", "item3")));
-    assertThat(pushItems.toString(), equalTo("PushItems [items=[[itemId=item1, pushItem={}], "
-            + "[itemId=item2, pushItem={queue=test queue}], [itemId=item3, pushItem={}]]]"));
+            .map(item -> item.getId()).collect(ImmutableList.toImmutableList()),
+            equalTo(ImmutableList.of("item1", "item2", "item3")));
+    assertThat(pushItems.toString(), equalTo("PushItems [items=[[itemId=item1, pushItem=GenericData{classInfo=[contentHash, metadataHash, payload, queue, repositoryError, structuredDataHash, type], {}}], [itemId=item2, pushItem=GenericData{classInfo=[contentHash, metadataHash, payload, queue, repositoryError, structuredDataHash, type], {queue=test queue}}], [itemId=item3, pushItem=GenericData{classInfo=[contentHash, metadataHash, payload, queue, repositoryError, structuredDataHash, type], {}}]]]"));
```

Comment:
Are we changing the response for the PushItem due to some parser upgrade? Why was this needed? Which dependency change caused this?

Reply:
Due to update in api version (not sure which one exactly) there was a change in PushItems response string.